packages/fluentui/accessibility/src/behaviors/Image/imageBehavior.ts

```diff
@@ -15,6 +15,12 @@ export const imageBehavior: Accessibility<ImageBehaviorProps> = props => ({
   },
 });
 
+export function useImageAria(props: ImageBehaviorProps): AccessibilityAttributes {
+  return {
+    'aria-hidden': props.alt || props['aria-label'] ? undefined : 'true',
+  };
+}
```

Comment:
I don't think that it's an expected way to go. @mnajdova do you have some input on this?

Reply:
Ideally I think we like to re-use the structure we had before, as it's clear by what we had so far we need to define aria props for all the slots inside. 

For the key-handlers, still not sure, as I am still looking over those. Not sure if we should make them part of the accessibility or just add them to the `useImage` hook directly.. 

Anyway, I would at least propose adding these inside `root`, and then if we extend the APIs we can add attributes as a first key.

Reply:
After some thoughts, I think we should add these to the `useButtonState` hook. I really don't see a value for us learning to write yet another hook, when already some of the props & key handlers are inside this hook. We can work more towards testing the a11y better, but from implementation point of view, I think it's easier to just include these things inside the `useButtonState` hook. What are your thoughts?

Reply:
Changes to accessibility will be done in #14506.